omero/users/cli/import-target.txt

```diff
@@ -133,3 +133,48 @@ are under::
 For more information on the regular expression syntax that can be used in
 templates see:
 `java.util.regex.Pattern documentation <http://docs.oracle.com/javase/7/docs/api/java/util/regex/Pattern.html>`_.
+
+Importing to targets across groups
+----------------------------------
+
+Currently, in all the above cases the import target must be in the user's
+current group for the import to succeed. It is hoped that this limitation can be
+removed in a later version of OMERO. This is also pertinent if the target is
+likely to be created as it will be created in the current group, which may not
+be the group intended.
+
+If no group is specified by using the :option:`-g` login option as part of the
+import, the current group will depend on whether the user is currently logged
+in, is logged out but has active sessions or is logged out with no active
+sessions. If the user is logged in then their current group will be the one they
+are logged in to. If the user is logged out but has active sessions then the
+most recent session will be used to connect and that will determin the current
+group. Finally, if the user is logged out and has no active sessions then the
+current group will be their default group.
```

Comment:
trying to decide if this paragraph could be better as a three-bullet list or table of cases or something